### PR TITLE
Conn 1196

### PR DESCRIPTION
**Pull request by alameluchidambaram**

### CODE REVIEW

File: Product/Production/Adapters/General/CONNECTDirectConfig/src/main/webapp/WEB-INF/applicationContext.xml [hunk 1]
```diff
@@ -23,24 +23,4 @@
...
     <jaxws:endpoint id="configurationService" address="/ConfigurationService" implementor="gov.hhs.fha.nhinc.directconfig.service.impl.ConfigurationServiceImpl" />
 
-    <!-- Camel Context for cache update and trust bundle processors -->
-    <bean id="bundleRefreshProcessor" class="gov.hhs.fha.nhinc.directconfig.processor.impl.DefaultBundleRefreshProcessorImpl" />
-
-    <bean id="bundleCacheUpdateProcessor" class="gov.hhs.fha.nhinc.directconfig.processor.impl.DefaultBundleCacheUpdateProcessorImpl">
-        <property name="refreshProcessor" ref="bundleRefreshProcessor" />
-    </bean>
-
-    <camelContext id="web-camel-context" xmlns="http://camel.apache.org/schema/spring">
```
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTDirectConfig/src/main/webapp/WEB-INF/applicationContext.xml:33]: Can you also remove the unused schema location?

File: Product/Production/Deploy/direct-dependencies/pom.xml [hunk 2]
```diff
@@ -110,31 +110,6 @@
         </dependency>
 
         <dependency>
-            <groupId>org.bouncycastle</groupId>
```
Comment (christophermay07) [inline Product/Production/Deploy/direct-dependencies/pom.xml:113]: We still need the bouncycastle dependency.

File: Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/processor/impl/DefaultBundleRefreshProcessorImpl.java [hunk 3]
```diff
@@ -199,7 +198,6 @@ public boolean verify(String hostname, SSLSession session) {
      * {@inheritDoc}
      */
     @Override
-    @Handler
```
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/processor/impl/DefaultBundleRefreshProcessorImpl.java:202]: For DefaultBundleCacheUpdateProcessorImpl, we have the same Handler tag and import that you removed here, only commented out -- can you clean that up for consistency?

File: Product/Production/Adapters/General/CONNECTDirectConfig/src/main/webapp/WEB-INF/applicationContext.xml (hunk 1 above)
Comment (alameluchidambaram) [inline Product/Production/Adapters/General/CONNECTDirectConfig/src/main/webapp/WEB-INF/applicationContext.xml:33]: Removed the unused Apache Camel schema in the latest commit.

File: Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/processor/impl/DefaultBundleRefreshProcessorImpl.java (hunk 3 above)
Comment (alameluchidambaram) [inline Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/processor/impl/DefaultBundleRefreshProcessorImpl.java:202]: Removed the @Handler annotation and the corresponding import.

Comment (alameluchidambaram) [inline Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/processor/impl/DefaultBundleRefreshProcessorImpl.java:202]: The bouncycastle dependency is back in the codebase.

**Pull request merged**